paasta_tools/api/api.py

```diff
@@ -102,6 +101,7 @@ def setup_paasta_api():
 
 
 def main(argv=None):
+    monkey.patch_all()
```

Comment:
@matthewbentley moving this is okay right? One of the problems I was having with the example cluster seemed to be related to the monkey patching. Something to do with using pserve to launch the API server which means we're not in the main thread...

Anyway, this should mean we wont have the monkey patching in the example cluster but we will still have it in production.